src/scheduler/workers/processListsBeforeAndDuringStart/helpers.ts

```diff
@@ -33,9 +33,14 @@ export function isEmailSentBefore(
     sendStartedPostEmail: ["sendStartedPostEmail"],
     sendStartedProviderEmail: ["sendStartedProviderEmail"],
   };
-  const listEventJsonData = audit.jsonData as ListEventJsonData;
   const subsequentEmails: string[] = subsequentEmailsForReminderType[reminderType];
-  const reminderHasBeenSent = subsequentEmails?.includes?.(listEventJsonData?.reminderType as string) ?? false;
+  let reminderHasBeenSent = false;
+  if ("createdAt" in event) {
+    reminderHasBeenSent = subsequentEmails?.includes?.((event.jsonData as ListEventJsonData)?.reminderType as string);
+  } else {
+    reminderHasBeenSent = subsequentEmails?.includes?.((event.jsonData as Record<string, string>)?.notes);
```

Comment:
notes is an array - so it would be doing something like `subsequentEmails.includes(['a', 'b'])`. Is that right? 

if it's easier, we can do another migration to add reminderType to the event json (along side having notes). 

Have you checked the DB? are you able to find `sendStartedProviderEmail` in the event table?

Reply:
TO my knowledge the notes key should usually has an array with just one value so should this be okay `(event.jsonData as Record<string, string>)?.notes[0]`?

Reply:
Yep in the DB, `sendStartedProviderEmail` is there under notes. Here's an example jsonData 
```json
{ "notes": ["sendStartedProviderEmail"], "eventName": "reminder", "reference": "xxx" }
```

Although `sendStartedProviderEmail` didn't exist in the Events table before my migration script ran.